docs/api/create_docker_container.md

```diff
@@ -2,6 +2,36 @@
 
 Testcontainers' generic container support offers the greatest flexibility and makes it easy to use virtually any container image in the context of a temporary test environment. To interact or exchange data with a container, Testcontainers provides  `TestcontainersBuilder<TDockerContainer>` to configure and create the resource.
 
+## Configure container start
+
+Both `ENTRYPOINT` and `CMD` allows you to configure an executable, that a container runs at the start. By default a container will run whatever `ENTRYPOINT` or `CMD` is specified in the Docker container image. At least one of both configurations is necessary. The container builder implementation supports `WithEntrypoint(params string[])` and `WithCommand(params string[])` to set or override the executable. The `ENTRYPOINT` should set the containers executable, where the `CMD` set the default arguments for an `ENTRYPOINT`.
+
+Instead of running NGINX, the following container configuration overrides the default start and just test the NGINX configuration file.
+
+```csharp
+_ = new TestcontainersBuilder<TestcontainersContainer>()
+  .WithEntrypoint("nginx")
+  .WithCommand("-t")
+```
+
+## Configure container app or service
+
+Apps or services running inside a container are usually configured either with environment variables or configuration files. `WithEnvironment(string, string)` sets an environment variable, while `WithResourceMapping(string, string)` copies a file into a container before it starts. This covers common use cases among many .NET applications.
+
+!!!tip
+
+    The majority of builder methods are overloaded and have different parameters to set configurations.
+
+To configure an ASP.NET Core application, either one or both can be used.
+
+```csharp
+_ = new TestcontainersBuilder<TestcontainersContainer>()
+  .WithEnvironment("ASPNETCORE_URLS", "https://+")
+  .WithEnvironment("ASPNETCORE_Kestrel__Certificates__Default__Path", "/app/certificate.crt")
+  .WithEnvironment("ASPNETCORE_Kestrel__Certificates__Default__Password", "password")
+  .WithResourceMapping("certificate.crt", "/app/certificate.crt")
```

Comment:
Should not be also added line with `.WithBindMount("...","...")` to example as alternative to `.WithResourceMapping(...)`?

Reply:
I didn't add it, because we do not recommend it. It makes test difficult to set up among different developer machines and CI pipelines (due to different file paths). Since we have the resource mapping I would promote it as best practice.

Mentioning it as an alternative that explains the drawbacks might be still a good idea. I can add it.

Reply:
I've mentioned `WithBindMount(string, string)` below.